### ISSUE TITLE
BODYMOVIN WILL NOT INSTALL

### ISSUE THREAD

**Issue by jorgeluisvasconez** [aescripts/aescripts-updater-public#4]
Running AE on the most recent version. I've downloaded aescripts manager.

When I go to download Bodymovin it says no compatible host applications found. When i perform a custom install it doesn't start

**Comment by Maxweel** [comment]
Hi,

    are you on Mac or Windows? Which versions of AE do you have installed?

Kevin

2017-07-24 16:03 GMT+02:00 jorgeluisvasconez <notifications@github.com>:

**Comment by jorgeluisvasconez** [comment]
Working on a Mac. Version 14.2.0.198

I ended up downloading a zxp installer to make it work.

Could this just be a bug with the beta app?

**Comment by Maxweel** [comment]
Yes I just checked, I think it's just a bug in the ZXP installation. You
were right to use the ZXP Installer in the end, and thanks for telling me
about this.

Thanks,

Kevin
